ietf/secr/urls.py

```diff
@@ -6,6 +6,5 @@
     re_path(r'^announcement/', include('ietf.secr.announcement.urls')),
     re_path(r'^meetings/', include('ietf.secr.meetings.urls')),
     re_path(r'^rolodex/', include('ietf.secr.rolodex.urls')),
-    re_path(r'^sreq/', include('ietf.secr.sreq.urls')),
```

Comment:
This should be a permanent RedirectView to the new location for at least one meeting cycle